### DISCUSSION TITLE
Is Whisper asking for sponsorship?

### DISCUSSION THREAD

**Discussion by ioudove** [openai/whisper#1724] [Q&A] [answered]
I took a m4a file of 3:49:01 and converted it to srt. srt file shows that from 03:24:29,640 onwards, all the speeches are "Please do not begrudge me for liking, subscribing, forwarding, and rewarding to support the Mingjing and DotDot columns (in Chinese)". Is this a bug?

**Comment by ioudove** [comment]
Addendum: Whisper stopped working before my video had finished playing. I'm guessing it might be because there was a period of time in the video where no one was talking.

**Comment by ryanheise** [comment, marked as answer]
This is called a hallucination. i.e. whisper is hallucinating words that were never actually spoken. It also tends to happen when there is silence. Since this has been discussed before, I would suggest searching the discussions for "hallucinations" to learn about the state of things and what you can do to minimise hallucinations.
